models/mednist_reg/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.0.4",
+    "version": "0.0.5",
     "changelog": {
```

Comment:
Do we need update the monai version for this bundle?

Reply:
Ok, but I guess the ci have already test with the latest monai version, so the bundle can work with the new version, right?

Reply:
for each bundle, CI uses it's corresponding monai/torch/... versions.